api/envoy/config/metrics/v2/stats.proto

```diff
@@ -184,6 +184,11 @@ message StatsdSink {
   string prefix = 3;
 }
 
+// can it be empty?
```

Comment:
Technically, yes, but then it might make sense to not provide a config at all by using this implementation in your sink and not add a configuration to the API at all:
```
ProtobufTypes::MessagePtr HystrixSinkFactory::createEmptyConfigProto() {
    return ProtobufTypes::MessagePtr{new Envoy::ProtobufWkt::Empty()};
  }
```

Reply:
I thought of having the number of buckets as a parameter to the constructor. And theoretically it can be changed even during action.  But not sure if this level of control is really necessary.